save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -535,14 +536,15 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                     alignItems = AlignItems.center
                 }
                 label {
+                    className = ClassName("btn")
+                    title = "Change organization's avatar"
                     input {
                         type = InputType.file
                         hidden = true
                         onChange = { event ->
                             postImageUpload(event.target)
                         }
                     }
```

Comment:
Suggestion to add a tooltip

Reply:
The 'title' here displays a tooltip when you hover over it with the mouse. Or did you mean something else?

Reply:
Yes, I meant `bootstrap`s tooltip, not the usual one (with data-toggle = tooltip). Is it here?

Reply:
Added screnshoot

Reply:
Yep, this is not `bootstrap` tooltip, to my mind it looks not that good. Check [this](https://getbootstrap.com/docs/4.6/components/tooltips/) out and find examples in our code (useTooltip hook can be used only inside `FC`s so either replace the hook with some code or use `(V)FC` wrapper).

Reply:
As discussed above, will be done in next PR